### PR TITLE
Feat/organize mvp

### PR DESCRIPTION
**Pull request by vdedios**
This is a mini restructure over [George's previous mvp work](https://github.com/42AGV/webserv/pull/12). The code remains the same but unused files have been removed and folder structure has been added

### CODE REVIEW

File: srcs/app/server/Location.cpp [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#include <Location.hpp>
+
+Location::Location(const CommonConfig &common) : common(common) {
```
Comment (gbudau) [inline srcs/app/server/Location.cpp:3]: I think this file can go in  `srcs/app/config/` because it's a component of the `ServerConfig` class

**Review by gbudau — APPROVED** [review]
CommonConfig and ServerConfig makes more sense :+1:
LGTM! :tada: :rocket:

File: srcs/app/server/Location.cpp (hunk 1 above)
Comment (vdedios) [reply srcs/app/server/Location.cpp:3]: You are completely right! I will fix it for sure, thanks!!

**Review by Damrod — APPROVED** [review]
Good!

**Pull request merged**